hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/auth/HmppsResourceServerConfiguration.kt

```diff
@@ -29,7 +28,6 @@ import uk.gov.justice.hmpps.kotlin.auth.dsl.ResourceServerConfigurationCustomize
 @ConditionalOnWebApplication(type = SERVLET)
 @EnableWebSecurity
 @EnableMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
-@EnableCaching
```

Comment:
Not needed anymore, can caused an issue in tests

Reply:
Do we still need `org.springframework.boot:spring-boot-starter-cache` then too?  Might run into issues if projects have removed `@EnableCaching` thinking the lilbrary was going to do it for them...

Reply:
Yes we should remove, and yes we will have a breaking change. Luckily the library is only implemented in our team's proejcts so far.